webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.js

```diff
@@ -59,8 +61,15 @@ const TaxonomyDropdown = ({ taxonomyType, currentTaxonomy, taxonomies }) => {
       }}
       isDisabled={!currentTaxonomy}
     >
-      {!currentTaxonomy && selectedIcon}
-      {anyTaxonomyText}
+      <Grid hasGutter>
+        <GridItem span={1}>
+          <i className="fa fa-globe" aria-hidden="true" />
```

Comment:
Can you use `<GlobeIcon />` here? its in `@patternfly/react-icons`.
Full list of icons is here: https://patternfly-react.surge.sh/icons/

Reply:
The icons search in https://patternfly-react.surge.shworth nothing lol
thanks for finding this!